buenavista/postgres.py

```diff
@@ -1,4 +1,5 @@
 import datetime
+import dateutil.parser
```

Comment:
Just need to add this dep into dev-requirements.txt and we will be good to go

Reply:
Done (and rebased)